Titles changes

comma opened the pull request:
Swiped TG's Cyberboss's implementation, now they actually roll.
Clients get verb to remove them.
Blackouts screen now for imporoved visibility.
Fixes monkeys sneaking into memorial section.
Randomizes last disclaimer bit slightly.
Other minor stuff in generating actual titles.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

PsiOmegaDelta commented on code/game/movietitles.dm:
```diff
@@ -22,7 +99,7 @@
 	possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "THE LITTLEST ARMALIS", "WHAT HAPPENS WHEN YOU MIX MAINTENANCE DRONES AND COMMERCIAL-GRADE PACKING FOAM", "ATTACK! ATTACK! ATTACK!", "SEX BOMB")]"
 	possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELIQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
-
+	possible_titles += "[pick("THE DAY TORCH STOOD STILL", "HUNT FOR THE GREEN WEENIE", "ALIEN VS VENDOMAT", "SPACE TRACK")]"
```
What if Torch isn't the current map?

comma replied:
Whoops

Techhead0 commented on code/game/movietitles.dm:
```diff
@@ -1,14 +1,91 @@
+	set category = "OOC"
+	verbs -= /client/proc/ClearCredits
+	QDEL_NULL_LIST(credits)
+	credits = null
```
Null it twice, just to make sure.

comma committed:
Titles changes

Swiped TG's Cyberboss's implementation, now they actually roll.
Clients get verb to remove them.
Blackouts screen now for imporoved visibility.
Fixes monkeys sneaking into memorial section.
Randomizes last disclaimer bit slightly.
Other minor stuff in generating actual titles.

Sunbeamstress commented on code/game/movietitles.dm:
```diff
@@ -34,46 +110,63 @@
 		titles += "<center>BASED ON REAL EVENTS<br>In memory of [english_list(corpses)].</center>"
 
 	var/list/staff = list("PRODUCTION STAFF:")
+	var/list/staffjobs = list("Coffe Fetcher", "Cameraman", "Angry Yeller", "Chair Operator", "Choreographer", "Historical Consultant", "Costume Designer", "Chief Editor", "Executive Assistant")
```
"Coffe" should be "coffee".

PsiOmegaDelta replied:
"coffee Fetcher"?

PsiOmegaDelta replied:
I see now, lack of letters.
